projects/sdk/src/lib/silo.ts

```diff
@@ -427,7 +454,7 @@ export class Silo {
    */
   async getSeeds(_account?: string) {
     const account = await Silo.sdk.getAccount(_account);
-    return Silo.sdk.contracts.beanstalk.balanceOfSeeds(account).then((v) => Silo.sdk.tokens.SEEDS.fromBlockchain(v));
+    return Silo.sdk.contracts.beanstalk.balanceOfLegacySeeds(account).then((v) => Silo.sdk.tokens.SEEDS.fromBlockchain(v));
```

Comment:
TO DISCUSS: This now reads from the legacy `balanceOfSeeds` storage slot:

```
function balanceOfSeeds(address account) internal view returns (uint256) {
        AppStorage storage s = LibAppStorage.diamondStorage();
        return s.a[account].s.seeds;
    }
```
    
Seeds don't really exist now. A Seed previously granted 1 / 10_000 Stalk per Season. Now each token has a configuration value at `s.ss[token].stalkEarnedPerSeason` which defines the amount of STALK earned per BDV per Season. We should discuss UX implications